paddle/fluid/pybind/eager_method.cc

```diff
@@ -961,6 +1015,15 @@ static PyObject* tensor_method_detach(TensorObject* self,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+PyDoc_STRVAR(tensor_method_detach___doc__, R"DOC(detach_($self, /)
+--
+Detach self from the current graph, and returns self Tensor.
```

Comment:
```suggestion

Detach self from the current graph, and returns self Tensor.
```
可能得空行吧，预览有些错
<img width="841" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/70642955/5985cefd-8943-404e-b80a-e1823e9992cd">